pytezos/block/header.py

```diff
@@ -65,13 +91,13 @@ def activate_protocol(cls, protocol_hash: str, parameters: Dict[str, Any], conte
     def bake_block(cls, context: ExecutionContext, min_fee: int = 0) -> 'BlockHeader':
         pending_operations = context.shell.mempool.pending_operations()  # type: ignore
         operations: List[Dict[str, Any]] = [
-            op
-            for op in pending_operations['applied']
-            if sum(map(lambda x: int(x['fee']), op['contents'])) >= min_fee  # handle batch case
+            op for op in pending_operations['applied'] if sum(map(lambda x: int(x['fee']), op['contents'])) >= min_fee  # handle batch case
         ]
         protocol_data = {
-            "priority": 0,  # default
-            "proof_of_work_nonce": "0000000000000000"  # dummy nonce
+            # FIXME: Default
+            "priority": 0,
+            # FIXME: Dummy nonce, won't work outside of the sandbox
+            "proof_of_work_nonce": "0000000000000000",
```

Comment:
It's ok for preapply, the actual real pow_nonce can be obtained by calling `.work()`